src/main/java/seedu/storage/ModStorage.java

```diff
@@ -115,6 +114,13 @@ public static void showModOffline(String moduleCode) {
         TextUi.printLocalSearchMessage();
     }
 
+    public static void addTimetableFile(Module module) throws IOException {
+        String filePath = "timetable/" + module.getModuleCode() + ".json";
```

Comment:
Should we move this to data/timetable to consolidate all of our saved data in one directory instead?

Reply:
I think @xuanlc113 already implemented a timetable saving method so im not sure this method is necessary

Reply:
In that case can this method be removed without breaking the code? @Poopies99 might also need to resolve the merge conflict due to @aditichadha1310's PR that was merged earlier.